server/src/useCases/report/requestUtils.ts

```diff
@@ -1,4 +1,4 @@
-const mapQueryToSorting = (queryParams, queryToMongoMapping) => {
+const mapQueryToSorting = (queryParams: any, queryToMongoMapping: Record<string, string>) => {
```

Comment:
can you define handled query params?

Reply:
i will define it but keep it for later, issue is for sorting parameters which should be defined as simple string but those are present in code as ParsedQs which allows you to use more than simple string.